pipeline_steps/kibana.groovy

```diff
@@ -8,7 +8,7 @@ def kibana(vm=null){
   common.conditionalStage(
     stage_name: "Kibana Tests",
     stage: {
-      kibana_tests(vm)
+      kibana_tests(branch, vm)
     }
   )
 }
```

Comment:
What about kibana_prep? The requirements that get installed into the venv are always from the master branch. By the looks of it they are currently the same but that is not guaranteed.